src/binomial.jl

```diff
@@ -0,0 +1,103 @@
+function stirlerr(n::Int64)
+    # stirlerr(x) = log(x!) -log(sqrt(2*pi*x)*(n/e)^n)
+    const S0 = 0.083333333333333333333        # 1/12
+    const S1 = 0.00277777777777777777778      # 1/360
+    const S2 = 0.00079365079365079365079365   # 1/1260
+    const S3 = 0.000595238095238095238095238  # 1/1680
+    const S4 = 0.0008417508417508417508417508 # 1/1188
+    # 35 values, sfe(x) = log(n!*e^n/((n^n)*sqrt(2*pi*n)))
+    sfe = (0.0810614667953272582196702, 0.0413406959554092940938221,  0.02767792568499833914878929,

```

Comment:
The comment should read '15', not '35'.  Also you use `sfe(x)` but the expression is in terms of `n`

A more Julian approach would be to create and store `sfe` as a `const` vector in the package, not in the function.  You could write it as
```julia
julia> const Stirlingerror = Float64[log(factorial(n)) - log(sqrt(2pi * n) * (n/Base.e)^n) for n in big(1):big(20)]
20-element Array{Float64,1}:
 0.0810615 
 0.0413407 
 0.0276779 
 0.0207907 
 0.0166447 
 0.0138761 
 0.0118967 
 0.0104113 
 0.00925546
 0.00833056
 0.00757368
 0.00694284
 0.00640899
 0.00595137
 0.00555473
 0.00520766
 0.0049014 
 0.00462915
 0.00438556
 0.00416632

julia> Stirlingerror[1]
0.08106146679532727
```
The calculation will be done in 256 bit precision then rounded to the 52 bits of precision for Float64 so it should be accurate.

This evaluation will be done at compile time.  If you change the condition for the explicit value branch in the code to
```julia
x < length(Stirlingerror) && return Stirlingerror[Int(x)]
```
then you can change the number of explicit values easily, in case you want to experiment with a cutoff.

Also, I would write the coefficients as `inv(12)` etc. if you want to use them as multiples.  It probably makes more sense to write the polynomial in terms of the inverse of the coefficients though.

Reply:
I made a mistake. The condition should be
```julia
x  ≤ length(Stirlingerror) && return Stirlingerror[Int(x)]
``